ldi-core/ldes-client-v2/request-executor/src/test/resources/features/default-request-executor.feature

```diff
@@ -31,4 +31,10 @@ Feature: DefaultRequestExecutor
     And I add a RequestHeader with key "If-None-Match" and value the obtained etag
     And I create a Request with the RequestHeaders and url: "https://grar.smartdataspace.dev-vlaanderen.be/addresses/by-name?substring=a"
     And I execute the request
-    Then I obtain a response with status code 304
\ No newline at end of file
+    Then I obtain a response with status code 304
+
+  Scenario: Obtaining a HttpRequestException of an invalid Request
+    Given I have a RequestExecutor
+    When I create RequestHeaders
+    And I create a Request with the RequestHeaders and url: "https://not-real"
+    Then I obtain a HttpRequestException when executing the request
```

Comment:
Personal preference probably.. but I find obtain a strange word to use in the context. 

It seems like "I get a ...." might be better in my eyes.
Obtaining sounds like a more active word than get. And in this context, the actions has already been performed, you're just getting it back. 

But like I said, personal preference. "des goûts et des couleurs, on ne discute pas" 😄

Reply:
I had some mixed feelings about obtain as well and went for it to stay consistent with the happy flows but for your lovely French quote, I will change it to 'get' :smile: